raiden/tests/unit/test_channelstate.py

```diff
@@ -1302,3 +1306,110 @@ def test_channelstate_unlock():
         settle_block_number,
     )
     assert must_contain_entry(iteration.events, ContractSendChannelBatchUnlock, {})
+
+
+def test_channel_unlock_must_not_change_merkletree():
```

Comment:
this test should not be merged, it doesnt make sense anymore

Reply:
Sure, removing it